### PR TITLE
[Merged by Bors] - Expanded Msaa documentation.

### LINKED ISSUE #3167: Document the behavior of the `Msaa` resource
## How can Bevy's documentation be improved?

[Current docs](https://docs.rs/bevy/0.5.0/bevy/prelude/struct.Msaa.html) are not very useful. They should:

- expand the acronym
- link to an external explanation of the anti-aliasing
- note the default values
- note the effect of changing the values
- demonstrate how to configure MSAA in your app with a simple example

### PR DESCRIPTION
**Pull request by Wcubed**
- Added default value.
- Links to the explanation of Msaa on wikipedia.
- Added a short example to show how to use the struct on an app.

Fixes #3167.

### CODE REVIEW

**Review by mockersf — APPROVED** [review]

File: crates/bevy_render/src/view/mod.rs [hunk 1]
```diff
@@ -40,9 +40,21 @@ impl Plugin for ViewPlugin {
 }
 
 #[derive(Clone)]
+/// Configuration resource for [Multi-Sample Anti-Aliasing](https://en.wikipedia.org/wiki/Multisample_anti-aliasing).
+///
+/// # Example
+/// ```
+/// # use bevy_render::prelude::Msaa;
```
Comment (mockersf) [inline crates/bevy_render/src/view/mod.rs:47]:
```suggestion
/// # use bevy_render::prelude::Msaa;
/// # use bevy_app::App;
```

Missing this import in the example

**Review by mockersf — CHANGES_REQUESTED** [review]

**Comment by Wcubed** [comment]
Good catch.
Interestingly, I ran `cargo run -p ci` locally, but it didn't error over that. It seems the local ci only runs `bevy_ecs_compile_fail_tests`. Is this intentional?

**Review by mockersf — APPROVED** [review]

**Comment by mockersf** [comment]
`cargo run -p ci` is meant to run the tests other that cargo test.

In Bevy CI:
- `cargo test --workspace` run on windows stable, macos stable, linux stable and linux nightly
- `cargo run -p ci` run on linux stable, as most of those (fmt, clippy) are platform independent anyway

**Pull request closed without merging**